package.json

```diff
@@ -4,8 +4,26 @@
   "description": "Full-stack document management system",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon --exec babel-node ./bin/www",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "nodemon  --exec babel-node ./bin/www",
```

Comment:
How about letting the start command be for production and, instead of what you have now, create a custom start command for development. Also, babel node should not be used in production and/or maybe nodemon as well.